SCClassLibrary/Common/Control/Server.sc

```diff
@@ -717,6 +718,14 @@ Server {
 		^Buffer.cachedBufferAt(this, bufnum)
 	}
 
+	defaultGroupIDForClientID { |id| ^nodeAllocator.numIDs * id + 1 }
+
+	sendDefaultGroupsFor { |ids|
+		ids.do { |id|
+			this.sendMsg("g_new", this.defaultGroupIDForClientID(id));
```

Comment:
better `"/g_new"`.

Reply:
it looks like this still needs to be changed